provider/provider.go

```diff
@@ -90,7 +91,16 @@ func getDefaultPath() string {
 			programFilesDir = path.Join("C:", "Program Files")
 		}
 
-		return path.Join(programFilesDir, "Cyberark Conjur", "Summon", "Providers")
+		dir := path.Join(programFilesDir, "Cyberark Conjur", "Summon", "Providers")
+
+		// enable portable installation with Providers dir next to executable
+		if _, err := os.Stat(dir); os.IsNotExist(err) {
+			exec, _ := os.Executable()
+            execDir := filepath.Dir(exec)
```

Comment:
Indentation is wrong here. Make sure you use spaces for all indentation here (or run the file through gofmt)